src/doom/p_enemy.c

```diff
@@ -235,33 +235,32 @@ boolean P_CheckMissileRange (mobj_t* actor)
 
     dist >>= FRACBITS;
 
-    if (actor->type == MT_VILE)
+    // [crispy] generalization of the Arch Vile's different attack range
+    if (actor->info->maxattackrange > 0)
     {
-	if (dist > 14*64)	
```

Comment:
Here and in line 248 is still whitespace diff.

Reply:
You mean you want to re-add the trailing whitespace at the end "	"?

Reply:
Nope, the indentation is still different.

Reply:
It seems the original uses a tab even though everything else around uses spaces.
Line 248 seems to originally combine 1 tab + 2 spaces.
I'll change it to match how it was before, although I don't think this would affect the patch size since it's an already modified line.